### PR TITLE
Upgrade to elixir 13.1/otp 24

### PR DESCRIPTION
**Pull request by BartoszMaj**
This change replace the old crypto method for hmac authorization that was removed in OTP 23.
Additionally Elixir version is updated to 13.1 along with the calendar library update to 1.0.

### CODE REVIEW

**Review by zelaznik — APPROVED** [review]

**Review by nialbima — APPROVED** [review]
My comments never posted! this is FINE. APPROVED.

File: lib/api_auth/authorization_header.ex [hunk 1]
```diff
@@ -42,8 +42,8 @@ defmodule ApiAuth.AuthorizationHeader do
   end
 
   defp signature(canonical_string, secret_key, algorithm) do
-    algorithm
-    |> :crypto.hmac(secret_key, canonical_string)
+    :hmac
+    |> :crypto.mac(algorithm, secret_key, canonical_string)
     |> Base.encode64()
   end
```
[the remark this answers is not in the document]
Comment (BartoszMaj) [reply lib/api_auth/authorization_header.ex:48]: In short: both of these do the exact same thing. `.hmac(algorithm,` was deprecated in favor of `.mac(:hmac,algorithm...`. The way it's called looks weird if you're not familiar with Elixir, but `a |> foo()` is equivalent to `foo(a)`.

File: .circleci/config.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: elixir:1.5
+      - image: elixir:1.13.1
```
[the remark this answers is not in the document]
Comment (BartoszMaj) [reply .circleci/config.yml:5]: Here's the image https://hub.docker.com/layers/elixir/library/elixir/1.13.1/images/sha256-23dee4bc885741a5bc3be58af6fb3dcc0d7e3b00245e09ffb3eb7d9e9dae58f7?context=explore
But I have no clue where to find release notes

[the remark this answers is not in the document]
Comment (BartoszMaj) [reply .circleci/config.yml:5]: CircleCI built it just fine

**Pull request merged**